app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def latest_posts
+    Post.where(author: self).limit(3)
+  end
+end
```

Comment:
- [ ] Kindly make sure the `User` model has validations, it should contain the following validations:
   - [ ] the `name` must not be blank
   - [ ] The `posts_counter` must be an integer greater than or equal to zero

Reply:
done